WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -495,8 +495,6 @@ public void StartCoinjoinSweeping(string? password = null, string? outputWalletN
 
 	private async Task StartCoinjoinSweepAsync(Wallet activeWallet, Wallet outputWallet)
 	{
-		activeWallet.ConsolidationMode = true;
```

Comment:
Why does this have been removed?

Reply:
Once you start the sweep this will never set back to false. Even if you just do normal coinjoin with the same wallet. This was a bug IMO. Other then that, now we are only changing the boolean at one place with a more or less clear log - deep inside the business logic "automatically".